crypto/bn/bn_test.cc

```diff
@@ -655,50 +654,6 @@ static bool TestHex2BN() {
   return true;
 }
 
-static bool TestRand(RAND *rng) {
-  ScopedBIGNUM bn(BN_new());
-  if (!bn) {
-    return false;
-  }
-
-  // Test BN_rand accounts for degenerate cases with |top| and |bottom|
-  // parameters.
-  if (!BN_rand(bn.get(), 0, 0 /* top */, 0 /* bottom */, rng) ||
-      !BN_is_zero(bn.get())) {
-    fprintf(stderr, "BN_rand gave a bad result.\n");
-    return false;
-  }
```

Comment:
It seems a little silly to me to test a test library that's going away soonish, but your call.